modules/Kconfig.tfm

```diff
@@ -38,3 +38,18 @@ config TFM_KEY_FILE_NS
 	  The path and filename for the .pem file containing the private key
 	  that should be used by the BL2 bootloader when signing non-secure
 	  firmware images.
+
```

Comment:
The Zephyr module menu starts to look quite messy, could you please apply the following patch.
```
diff --git a/modules/Kconfig.tfm b/modules/Kconfig.tfm
index a15f092da1..4d40d275d8 100644
--- a/modules/Kconfig.tfm
+++ b/modules/Kconfig.tfm
@@ -3,7 +3,7 @@
 # Copyright (c) 2019, 2020 Linaro Limited
 # SPDX-License-Identifier: Apache-2.0
 
-config BUILD_WITH_TFM
+menuconfig BUILD_WITH_TFM
        bool "Build with TF-M as the Secure Execution Environment"
        depends on TRUSTED_EXECUTION_NONSECURE
        help
@@ -21,6 +21,8 @@ config BUILD_WITH_TFM
            "mps2_an521_nonsecure") ensures that
            CONFIG_TRUSTED_EXECUTION_NONSECURE ie enabled.
 
+if BUILD_WITH_TFM
+
 config TFM_KEY_FILE_S
        string "Path to private key used to sign secure firmware images."
        depends on BUILD_WITH_TFM
@@ -53,3 +55,5 @@ config TFM_BL2_FALSE
         bool "TFM BL2 disabled"
 
 endchoice
+
+endif # BUILD_WITH_TFM
```

That will make TF-M related settings to have it's own menu in menuconfig looking like:
![image](https://user-images.githubusercontent.com/9802583/92218357-48fa6d80-ee99-11ea-8334-87a080ea79ae.png)
![image](https://user-images.githubusercontent.com/9802583/92218407-5ca5d400-ee99-11ea-8ddd-092368ef2645.png)

instead of:
![image](https://user-images.githubusercontent.com/9802583/92218464-7810df00-ee99-11ea-834c-73be0489692d.png)

Reply:
i've applied this patch